plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_tw.properties

```diff
@@ -17,6 +17,8 @@ dataTransfer.processor.xlsx.property.extension.label	=	\u6587\u4ef6\u64f4\u5c55\
 dataTransfer.processor.xlsx.property.header.description	=	\u5b57\u6bb5\u540d\u7528\u4f5c\u9996\u884c
 dataTransfer.processor.xlsx.property.header.name	=	\u5b57\u6bb5\u540d\u4f5c\u70ba\u5217\u6a19\u984c
 dataTransfer.processor.xlsx.property.headerFont.description	=	\u9996\u884c\u5b57\u9ad4\u5c6c\u6027
+dataTransfer.processor.xlsx.property.headerFontName.name=\u6a19\u984c\u5b57\u9ad4\u540d\u7a31\u000d\u000a
+dataTransfer.processor.xlsx.property.headerFontName.description=\u7b2c\u4e00\u884c\u5b57\u9ad4\u540d\u7a31
```

Comment:
Until you're sure that you know that language, such translations are useless and tend to contain grammatical errors.